app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java

```diff
@@ -244,6 +256,16 @@ public void onPause() {
         saveImmediate();
     }
 
+    @Override
+    public void onCreateOptionsMenu(final Menu menu, final MenuInflater inflater) {
+        if (DEBUG) {
+            Log.d(TAG, "onCreateOptionsMenu() called with: menu = [" + menu
+                    + "], inflater = [" + inflater + "]");
```

Comment:
I'd personally prefer formatting it something like:

```suggestion
            Log.d(TAG, "onCreateOptionsMenu() called with: "
                    + "menu = [" + menu + "], inflater = [" + inflater + "]");
```

Reply:
Fixed in 776dddd